llvmlite/llvmpy/core.py

```diff
@@ -243,11 +243,13 @@ def icmp(self, pred, lhs, rhs, name=''):
         else:
             return self.icmp_signed(_icmp_smap[pred], lhs, rhs, name=name)
 
-    def fcmp(self, pred, lhs, rhs, name=''):
+    def fcmp(self, pred, lhs, rhs, name='', flags=()):
```

Comment:
I wonder whether we should leave the llvmpy layer as-is for this PR rather than changing it, and instead:

- Swap all uses of `llvmpy.core.Builder` in Numba for `ir.IRBuilder` and replace the `icmp()` and `fcmp()` calls with the relevant ones (`icmp_unsigned()`, etc.)
- Remove the `Builder` class from here.

My thinking here is that llvmpy was only intended as a compatibility layer for Numba, and will eventually be removed (I did some Numba changes / removal in the past) - so we should move in that direction instead of "maintaining" it.